hnn_core/cell_response.py

```diff
@@ -370,6 +370,13 @@ def plot_spikes_hist(self, trial_idx=None, ax=None, spike_types=None,
             Valid strings also include leading characters of spike types
 
             | Ex: ``'ev'`` is equivalent to ``['evdist', 'evprox']``
+        color : str | list of str | None
+            String input defining color of histograms plotted. If list of str
+            provided, histograms for each cell type will be plotted by cycling
```

Comment:
```suggestion
            provided, histograms for each spike type will be plotted by cycling
```
Just to be consistent with our use of the `spike_types` argument.